web/src/frontend/src/digisos/redux/rest-utils.ts

```diff
@@ -31,6 +31,20 @@ export function fetchPost(urlPath: string, body: string) {
 		.then(toJson);
 }
 
+export function fetchPut(urlPath: string, body: string) {
+	const OPTIONS: RequestInit = {
+		headers: {
+			"Content-Type": "application/json"
+		},
+		method: "PUT",
```

Comment:
fetchPost er like, bare denne linjen som er forskjellig. Dra ut i en ny metode som disse to bare sender inn metoden til?

Reply:
Enig. Har slått sammen kode for å gjøre POST, PUT og GET til samme metode.